
Some stuff we can't do.

  - Rene Rebe <rene@exactcode.de>

--- rt2500-1.1.0/Module/Makefile.orig	2005-08-28 19:44:27.000000000 +0200
+++ rt2500-1.1.0/Module/Makefile	2005-08-28 21:55:39.000000000 +0200
@@ -106,12 +106,7 @@
 	@echo "install '$(MODULE_OBJECT)' to $(MODULE_ROOT)"
 	install -m 755 -o 0 -g 0 -d $(MODULE_ROOT)
 	install -m 644 -o 0 -g 0 $(MODULE_OBJECT) $(MODULE_ROOT)
-	/sbin/depmod -a
 	
-	@if ! grep -q 'ra0' /etc/modules.conf ; then \
-		echo "append 'alias ra0 rt2500' to /etc/modules.conf"; \
-		echo "alias ra0 rt2500" >> /etc/modules.conf ; \
-	fi
 
 install-fedora:
 	if ! [ -f $(MODULE_OBJECT) ]; then \
